Makefile.in

```diff
@@ -112,6 +123,30 @@ $(ARCHDIR)/%.lo: $(SRCDIR)/$(ARCHDIR)/%.c
 test: all
 	$(MAKE) -C test
 
+# The standalone fuzz target runner.
+standalone_fuzz_target_runner.o: $(SRCDIR)/test/fuzz/standalone_fuzz_target_runner.c
+	$(CC) $(CFLAGS) $(FUZZ_CFLAGS) $(INCLUDES) -c -o $@ $<
```

Comment:
Note: we need to compile all files of zlib-ng with the sanitizers `$(FUZZ_CFLAGS)`, not only the code of the fuzzer.

Reply:
We need to build two versions of the library: one without the sanitizers and one with.  The lib compiled with sanitizers will be used to link the fuzzers against when building the target `test`.

Reply:
Fixed in 80a5bab. I will update the current patch with this fix.